Automattic/woocommerce-payments
# [Authorizations] Wire Uncaptured Transactions screen to API data

This PR depends on #4949 

Fixes #4503 

#### Changes proposed in this Pull Request

This PR adds real data to the Uncaptured transactions screen (still behind a flag)

<img width="899" alt="Screen Shot 2022-10-21 at 14 44 16" src="https://user-images.githubusercontent.com/12542046/197210445-d3c9d484-1c49-4929-9e32-408005a03653.png">

This PR:
- Get authorizations data from the API and displays it in a table
- Make each row clickable and redirect to the payment details for this specific payment
- Displays a summary in the end of the table
- Displays total amount of authorizations (only if the same currency - multiple currency filtering is out of scope)

#### Testing instructions





* First things first, remove the "flag" by setting `displayAuthorizations=true` in `client/transactions/index.tsx`
* Add a few authorizations to the DB either manually, or from a real order (My recommendation is to create a real order, then copy-paste the data multiple times). Make sure to add them using the same currency.
* Open the `Transactions>Uncaptured` screen and double check things are working fine. A summary should be displayed with total authorizations and total amount
* Change the currency of one authorization (may need to remove the memcache docker instance too to destroy cache)
* Open the `Transactions>Uncaptured` screen again and double check the summary only displays how many authorizations available

-------------------


**Post merge**
